lib/src/store.dart

```diff
@@ -122,7 +122,7 @@ class Store extends Stream<Store> with Disposable {
   ///
   /// If the `Store` is disposing or has been disposed, this method has no effect.
   void trigger() {
-    if (isDisposedOrDisposing) return;
+    if (isOrWillBeDisposed) return;
```

Comment:
replace deprecated use of isDisposedOrDisposing with isOrWillBeDisposed

Reply:
Yeah, this breaks compatibility with the minimum ^1.7.0 versions of w_common. It was added in 1.9.0 so either 1) leave isDisposedOrDisposing or 2) update the minimum to 1.9.0
@evanweible-wf @georgelesica-wf preference?

Reply:
FWIW: I checked some top level repos and they are all already using w_common 1.11.1 .. so I prefer upping the minimum.

Reply:
Yeah we should raise the minimum